eventuate-tram-consumer-jdbc/build.gradle

```diff
@@ -4,7 +4,7 @@ dependencies {
     compile project(":eventuate-tram-consumer-common")
     compile project(":eventuate-tram-jdbc")
 
-    compile "io.eventuate.local.java:eventuate-local-java-jdbc:$eventuateLocalVersion"
+    compile "io.eventuate.cdc:eventuate-local-java-jdbc:$eventuateCdcVersion"
```

Comment:
Eventuate Tram (src/main)  should not depend on Eventuate CDC - SqlDialect should be moved into Common?

Reply:
I will fix